cfme/tests/webui/test_advanced_search.py

```diff
@@ -0,0 +1,113 @@
+    ('container_projects', 'All', 'container_projects', 'Container Project : Name'),
+    ('container_routes', 'All', 'container_routes', 'Container Route : Name'),
+    ('container_services', 'All', 'container_services', 'Container Service : Name'),
+    ('container_replicators', 'All', 'container_replicators', 'Container Replicator : Name'),
+    ('container_pods', 'All', 'container_pods', 'Container Pod : Name'),
+    ('containers', 'All', 'containers', 'Container : Name'),
+    ('container_nodes', 'All', 'container_nodes', 'Container Node : Name'),
+    ('container_volumes', 'All', 'container_volumes', 'Persistent Volume : Name'),
+    ('container_builds', 'All', 'container_builds', 'Container Build : Name'),
+    ('container_image_registries', 'All', 'image_registries', 'Container Image Registry : Name'),
+    ('container_images', 'All', 'container_images', 'Container Image : Name'),
+    ('container_templates', 'All', 'container_templates', 'Container Template : Name'),
+
+    (ConfigManager, 'All', 'configuration_management', 'Configuration Manager : Name'),
+    (ConfigSystem, 'All', 'configuration_management_systems',
+     'Configured System (Red Hat Satellite) : Hostname'),
+
+    ('network_providers', 'All', 'network_managers', 'Network Manager : Name'),
+    ('cloud_networks', 'All', 'network_networks', 'Cloud Network : Name'),
+    ('network_subnets', 'All', 'network_subnets', 'Cloud Subnet : Name'),
+    ('network_routers', 'All', 'network_routers', 'Network Router : Name'),
+    ('network_security_groups', 'All', 'network_security_groups', 'Security Group : Name'),
+    ('network_floating_ips', 'All', 'network_floating_ips', 'Floating IP : Address'),
+    ('network_ports', 'All', 'network_ports', 'Network Port : Name'),
+    ('balancers', 'All', 'network_load_balancers', 'Load Balancer : Name'),
+
+    ('volumes', 'All', 'block_store_volumes', 'Cloud Volume : Name'),
+    ('volume_snapshots', 'All', 'block_store_snapshots', 'Cloud Volume Snapshot : Name'),
+    ('volume_backups', 'All', 'block_store_backups', 'Cloud Volume Backup : Name'),
+
+    ('object_store_containers', 'All', 'object_store_containers',
+     'Cloud Object Store Container : Name'),
+    ('object_store_objects', 'All', 'object_store_objects',
+     'Cloud Object Store Object : Name'),
+
+    ('ansible_tower_providers', 'All', 'ansible_tower_explorer_provider',
+     'Automation Manager (Ansible Tower) : Name'),
+    ('ansible_tower_systems', 'All', 'ansible_tower_explorer_system',
+     'Configured System (Ansible Tower) : Hostname'),
+    ('ansible_tower_job_templates', 'All', 'ansible_tower_explorer_job_templates',
+     'Job Template (Ansible Tower) : Name'),
+    ('ansible_tower_jobs', 'All', 'ansible_tower_jobs', 'Ansible Tower Job : Name'),
+]
+
+pytestmark = [
+    pytest.mark.parametrize(
+        'params', params_values,
+        ids=['{}_{}'.format(param[2], param[1].lower()) for param in params_values]
```

Comment:
**REQUEST:** Can we please have ```-``` between the ids, it is the default for separating parameters.

Reply:
Done